html_table_parser/parser.py

```diff
@@ -52,7 +56,7 @@ def handle_endtag(self, tag):
             self._in_th = False
 
         if tag in ['td', 'th']:
-            final_cell = " ".join(self._current_cell).strip()
+            final_cell = "".join(self._current_cell).strip()
```

Comment:
I think the expected behavior of parsing the following table

```
<table>
  <tr>
    <td>
      <p>Hello</p>
      <p>World</p>
    </td>
  </tr>
</table>
```

would be `Hello World` instead of `HelloWorld`. Or do you think differently about that?